### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by PatrykMis** [NeoApplications/Neo-Store#295]
* "Sync" and "Settings" icons/buttons are not visible for TalkBack screen reader, eather by navigation gestures nor touch exploration (fixed in #309).
* Add 2-state switch or checkbox for each repository (e.g. like other switches in Settings). I cannot determine if repository is marked or unmarked unless I tap it one or two times: If I hear "connecting" then I know I've just enabled, and if not I've probably disabled it.
